bootstrap/roles/ocp4-install-noobaa/tasks/noobaa-create.yaml

```diff
@@ -0,0 +1,125 @@
+- name: Get cluster version
+  k8s_info:
+    api_version: config.openshift.io/v1
+    kind: ClusterVersion
+    name: version
+  register: r_cluster_version
+
+- name: Set ocp4_cluster_version fact
+  set_fact:
+    ocp4_cluster_version: "{{ r_cluster_version.resources[0].status.history[0].version }}"
+
+- name: Obtain Channel from Version
+  set_fact:
+    ocp4_channel: "{{ ocp4_cluster_version.split('.') }}"
+
+- name: Set Openshift Channel
+  set_fact:
+    ocp4_channel: "stable-{{ ocp4_channel[0] + '.' + ocp4_channel[1] }}"
+
+- name: Print OpenShift version
+  debug:
+    msg: "{{ ocp4_channel }}"
+
+- name: Adapt to the openshift_cluster_version LESS than 4.9
+  when: ocp4_cluster_version is version_compare('4.9', '<')
+  block:
+    - name: Create OpenShift Objects to install Noobaa
+      k8s:
+        state: present
+        definition: "{{ lookup('template', item ) | from_yaml }}"
+      loop:
+      - ./templates/odf-namespace.yaml.j2
+      - ./templates/operatorgroup-storage.yaml.j2
+      - ./templates/ocs-subscription.yaml.j2
+
+    - name: Wait for NooBaa CRD to exist
+      kubernetes.core.k8s_info:
+        api_version: "apiextensions.k8s.io/v1beta1"
+        kind: CustomResourceDefinition
+        name: "noobaas.noobaa.io"
+      register: crds
+      until: crds.resources|length > 0
+      retries: 30
+      delay: 10
+
+- name: Adapt to the openshift_cluster_version MORE than 4.9
+  when: ocp4_cluster_version is version_compare('4.9', '>=')
+  block:
+    - name: Create OpenShift Objects to install Noobaa
+      k8s:
+        state: present
+        definition: "{{ lookup('template', item ) | from_yaml }}"
+      loop:
+      - ./templates/odf-namespace.yaml.j2
+      - ./templates/operatorgroup-storage.yaml.j2
+      - ./templates/odf-subscription.yaml.j2
+
+    - name: Wait for NooBaa CRD to exist
+      kubernetes.core.k8s_info:
+        api_version: "apiextensions.k8s.io/v1"
+        kind: CustomResourceDefinition
+        name: "noobaas.noobaa.io"
+      register: crds
+      until: crds.resources|length > 0
+      retries: 30
+      delay: 10
+
+- name: Create Noobaa Object
+  k8s:
+    state: present
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+  - ./templates/noobaa-object.yaml.j2
+
+- name: Wait Until NooBaa Object is Ready
+  shell: |
+    oc get noobaas.noobaa.io/noobaa -n openshift-storage -o jsonpath='{.status.phase}'
```

Comment:
LGTM, in the future the k8s_info module can be use in order to have the possibility to not use oc directly within shell.